### ISSUE TITLE
Why doesn't the hashrate increase? miners are growing 

### ISSUE THREAD

**Issue by flockedsolid** [UnamSanctam/SilentCryptoMiner#1877]
Why doesn't the hashrate increase? miners are growing but the hashrate remains the same
I added the critical process option on crypter and use the binder with delay and the option to run as administrator, what am I doing wrong?

![1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/147938938/d3b30ed0-debb-424f-bdfc-e0b8e2d507b7)
![2](https://github.com/UnamSanctam/SilentCryptoMiner/assets/147938938/6e8e193c-475b-4eb0-bddb-aeff2d9b3435)

**Comment by UnamSanctam** [comment]
Well it looks like only one miner is online so that seems accurate doesn't it? Do you also have "Add Defender Exclusions" and "Disable Windows Update" disabled? Though since Microsoft recently added a new detection specifically targeting my miner (Trojan:Win64/SilentCryptoMiner) then that could possibly cause some of your miners to be removed.

**Comment by flockedsolid** [comment]
I left only one worker active so I don't get tangled up

**Comment by flockedsolid** [comment]
?

**Comment by UnamSanctam** [comment]
Either they are being removed by an antivirus, or you entered some "incorrect" settings, though it feels like the former is more likely since they have connected before. I would assume it's because of the new Trojan:Win64/SilentCryptoMiner detection.

**Comment by flockedsolid** [comment]
But how would it be possible for them to be deleted after being executed due to Critical Process, Disable, and all functions?

**Comment by flockedsolid** [comment]
![1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/147938938/dba05025-727f-4acf-b87f-b973ff11c92d)
![2](https://github.com/UnamSanctam/SilentCryptoMiner/assets/147938938/71cb0c38-c451-433e-9eab-148fe42411be)

**Comment by flockedsolid** [comment]
Is there a possibility of fixing this? Could you guide me on any line of code that I can change or redo to try to bypass this new check?
Is there any specific line that the antivirus detects that I could be messing with to try and fix it?

**Comment by UnamSanctam** [comment]
Not easily, the changes needed are very advanced, though I'm getting closer to finish the next version (was just able to bypass the newer compiler detections a minute ago).

**Comment by flockedsolid** [comment]
The Critical Process is not working. I even tested it on some machines here and none of them gave a critical process

**Comment by UnamSanctam** [comment]
It should work (I tested it earlier today), are you able to kill the processes? And do you have "Startup" (and "Run as System") enabled or not when testing it?

**Comment by flockedsolid** [comment]
yes, i have enabled everything, i can kill the processes and delete the file in %programfiles%

**Comment by UnamSanctam** [comment]
Yes you should be able to delete the files (the injected processes are critical), and you're able to kill the miner explorer.exe (or whatever you chose as the injection) and/or the watchdog injected into conhost.exe?

**Comment by flockedsolid** [comment]
the explorer.exe is empty, just the watchdog are working, what happen?

**Comment by UnamSanctam** [comment]
Not sure what you mean, what does it say when you run the checker on the machine that you're testing on?
